BoxContentSDK/BoxContentSDK/OAuth2/BOXParallelOAuth2Session.m

```diff
@@ -46,8 +46,8 @@ - (void)performRefreshTokenGrant:(NSString *)expiredAccessToken withCompletionBl
 }
 
 - (void)performRefreshTokenGrant:(NSString *)expiredAccessToken
-      newAccessTokenExpirationAt:(NSString *)accessTokenExpirationTimestamp
-     newRefreshTokenExpirationAt:(NSString *)refreshTokenExpirationTimestamp
+      newAccessTokenExpirationAt:(NSNumber *)accessTokenExpirationTimestamp
```

Comment:
Since the API documentation refers to this as a timestamp, we'll leave it this way for now to be consistent.